pil-analyzer/src/evaluator.rs

```diff
@@ -356,7 +356,7 @@ impl<'a, T: Display> Display for Value<'a, T> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
             Value::Bool(b) => write!(f, "{b}"),
-            Value::Integer(x) => write!(f, "{x}"),
+            Value::Integer(x) => write!(f, "{:#x}", x),
```

Comment:
This is for easier debugging, so I'm leaving it here until we are ready to have it merged.